src/test/resources/checkstyle.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE module SYSTEM "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
+<!DOCTYPE module PUBLIC
```

Comment:
The old one stopped downloading

Reply:
Hm, can we try to avoid downloading this schema?

```
<!DOCTYPE module [
        <!ELEMENT module (module|property|metadata|message)*>
        <!ATTLIST module name NMTOKEN #REQUIRED>

        <!ELEMENT property EMPTY>
        <!ATTLIST property
                name NMTOKEN #REQUIRED
                value CDATA #REQUIRED
                default CDATA #IMPLIED
                >

        <!ELEMENT metadata EMPTY>
        <!ATTLIST metadata
                name NMTOKEN #REQUIRED
                value CDATA #REQUIRED
                >

        <!ELEMENT message EMPTY>
        <!ATTLIST message
                key NMTOKEN #REQUIRED
                value CDATA #REQUIRED
                >
        ]>
```

Reply:
Maybe we can, the question is - should we?

Reply:
hm, prevent such issue in the future.. it is not PR blocker :)